README.md

```diff
@@ -1,88 +1,95 @@
+
+**Tech Stack**
 
-------
+- Ruby on Rails
 
-  **Tech Stack**
+---
 
-  - Ruby on Rails 
+**System Requirements**
+
+- [Ruby](https://www.ruby-lang.org/en/downloads/) version **3.0.0** or above.
+- [Imagemagick](https://imagemagick.org/) for image processing
 
 ---
 
-  **System Requirements**
+**Third Party Tools**
 
-  - [Ruby](https://www.ruby-lang.org/en/downloads/) version **3.0.0** or above.
-  - [Imagemagick](https://imagemagick.org/) for image processing
+- [Postmark](https://postmarkapp.com/) for transactional emails.
+- [AWS S3](https://aws.amazon.com/s3/) for Storage
+- [Facebook OAuth](https://developers.facebook.com/docs/facebook-login/web/) for Login with Facebook
+- [Google OAuth](https://developers.google.com/identity/protocols/OAuth2) for Login with Google
+- [Scout APM](https://www.scoutapm.com/) for Application Performance Monitoring.
 
 ---
 
-  **Third Party Tools**
+**Deployment**
 
-  - [Postmark](https://postmarkapp.com/) for transactional emails.
-  - [AWS S3](https://aws.amazon.com/s3/) for Storage
-  - [Facebook OAuth](https://developers.facebook.com/docs/facebook-login/web/) for Login with Facebook
-  - [Google OAuth](https://developers.google.com/identity/protocols/OAuth2) for Login with Google
-  - [Venter](https://github.com/VenterProject) (Machine Learning solution) for categorising responses.
+- The deployment is triggered via [Cloud66](https://www.cloud66.com/) app.
+- Upon building the app, a Cloud66 webhook is triggered which finally pushes the code to the server.
+- Servers are hosted on AWS and managed by [Cloud66](https://www.cloud66.com/).
 
 ---
 
-  **Deployment**
+**Credentials**
 
-  - The deployment is triggered via [Travis](https://travis-ci.com/) configured via the travis.yml file.
-  - Upon building the app, a Cloud66 webhook is triggered which finally pushes the code to the server. 
-  - Servers are hosted on Google Cloud and managed by [Cloud66](https://www.cloud66.com/).
+- Postmark API Key
+- AWS Access Key & Secret
+- Airbrake Project Key
+- Google Client ID and Secret
+- Facebook App ID and Secret
+- Scout Key
 
-----
+---
 
-  **Credentials**
+**Environment Variables**
 
-  - Postmark API Key
-  - AWS Access Key & Secret
-  - Sentry DSN link
-  - Google Client ID and Secret
-  - Facebook App ID and Secret
+- CLIENT_HOST
+- HOST
+- RAILS_MASTER_KEY
 
-----
+---
 
-  **Environment Variables**
+**Running locally**
 
-  - CLIENT_HOST
-  - HOST
-  - RAILS_MASTER_KEY
+- To run this project in localhost, do the following:
+  - `bundle install`
+  - `rails db:setup`
+  - `rails s`
+- To seed the database run `rails db:seed`
 
 ---
 
-  **Running locally**
+**Exploring GraphQL API**
 
-  - To run this project in localhost, do the following:
-    - `bundle install`
-    - `rails db:setup`
-    - `rails s`
-  - To seed the database run `rails db:seed`
+- Use GraphQL client such as [Postman](https://www.postman.com/) testing and interacting with the GraphQL API. .
+- GraphQL endpoint is at `/graphql`.
```

Comment:
@Mahaveer1141 - add an example for Staging and Production. 

Add a screenshot of how it looks on Postman GQL

Reply:
done